requirements.txt

```
@@ -0,0 +1,2 @@
+cryptography == 3.3.1
+requests == 2.25.1
```

```suggestion
cryptography==3.3.1
requests==2.25.1
```
These are the `requirements.txt` conventions.